data/builtInFeatures.js

```diff
@@ -28,7 +28,8 @@ const es2015 = {
   "es6.reflect.get-prototype-of": "Reflect / Reflect.getPrototypeOf",
   "es6.reflect.has": "Reflect / Reflect.has",
   "es6.reflect.is-extensible": "Reflect / Reflect.isExtensible",
-  "es6.reflect.own-keys": "Reflect / Reflect.ownKeys",
```

Comment:
So in pluginFeatures we would have something like https://github.com/babel/babel-preset-env/blob/master/data/pluginFeatures.js#L97-L99
```
  "transform-es2015-unicode-regex": {
    features: [
      'RegExp "y" and "u" flags / "u" flag, case folding',
      'RegExp "y" and "u" flags / "u" flag, Unicode code point escapes',
      'RegExp "y" and "u" flags / "u" flag',
    ],
  },
```

so I changed the format a bit but the logic doesn't work the same

```
const getLowestImplementedVersion = ({ features }, env) => {
  let tests = flatten(compatibilityTests
    .filter((test) => features.indexOf(test.name) >= 0)
```

same with DataView

so features would be ["typed arrays / DataView"]

["typed arrays / DataView"].indexOf(test.name) >= 0 where test.name === "typed arrays / DataView (Int32)".

I was thinking we wouldn't have to hardcode each test but that doesn't work.